index.d.ts

```diff
@@ -0,0 +1,126 @@
+/**
+ * Type definitions for Signale
+ */
+
+export interface BadgeColorText { badge?: String; color?: String; text?: Array<String> | String; }
+export interface TimesStart extends BadgeColorText { }
+export interface TimesEnd extends BadgeColorText { time?: String; }
+export interface Times { start?: TimesStart; end?: TimesEnd; }
+export interface Types { badge?: String, color?: String; label?: String; }
+export interface Stream { }
+export interface Config { displayScope?: Boolean; displayBadge?: Boolean; displayDate?: Boolean; displayFilename?: Boolean; displayLabel?: Boolean; displayTimestamp?: Boolean; underlineLabel?: Boolean; underlineMessage?: Boolean; underlinePrefix?: Boolean; underlineSuffix?: Boolean; uppercaseLabel?: Boolean; }
+export interface Options { times?: Times; types: Types; stream?: Stream; scope?: String; }
+export interface CompleteArrgumentType { prefix?: String; message: String; suffix?: String; }
```

Comment:
You have misspelt argument. Unless you're a pirate ☠️

Reply:
@jordrake Can you tell me which one I misspelled?

Reply:
`CompleteArrgumentType`

Reply:
@Cludch and @jordrake Thanks for pointing the issue to me. I fixed this.